test/test_scanner.py

```diff
@@ -16,6 +16,7 @@
 import unittest
 import unittest.mock
 from pathlib import Path
+from typing import List
 from test.utils import LONG_TESTS, download_file
```

Comment:
```suggestion
from test.utils import LONG_TESTS, download_file
from typing import List
```

Apparently I missed a line.